pyproject.toml

```diff
@@ -40,6 +40,7 @@ dependencies = [
     "opencv-python-headless>=4.0.0",
     "pandas",
     "pillow",
+    "paquo",
```

Comment:
we should make an "extra" to install paquo, because i don't think we should install it by default. paquo has a fairly heavy set of dependencies. look at the "dev" extra in the pyproject.toml file. i suggest you make another extra named "qupath" that installs paquo.

Reply:
Got it. I made an extra in the following way:

```toml
[project.optional-dependencies]
dev = [
    "black",
    "flake8",
    "geojson",
    "isort",
    "mypy",
    "pytest",
    "tiffslide",
    "types-Pillow",
    "types-tqdm",
    "Flake8-pyproject",
    "paquo"
]
docs = [
    "pydata-sphinx-theme",
    "sphinx",
    "sphinx-autoapi",
    "sphinx-click",
    "sphinx-copybutton",
]
openslide = ["openslide-python"]
qupath = ["paquo"]
```